MAUI/Calendar/date-restrictions.md

```diff
@@ -22,15 +22,15 @@ The [MinimumDateTime](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calend
 {% endhighlight %}
 {% highlight c# tabtitle="MainPage.xaml.cs" %}
 
-  this.Calendar.MinimumDateTime = DateTime.Now.AddDays(-5);
+  this.Calendar.MinimumDate = DateTime.Now.AddDays(-5);
```

Comment:
Check the property is MinimumDate or MinimumDateTime

Reply:
Checked the property, it is MinimumDate